platform/platform_tests/local_country_file_tests.cpp

```diff
@@ -112,17 +112,21 @@ UNIT_TEST(LocalCountryFile_DiskFiles)
 
     string const mapFileName = GetFileName(countryFile.GetName(), MapOptions::Map,
                                            version::FOR_TESTING_TWO_COMPONENT_MWM1);
-    ScopedFile testMapFile(mapFileName, ScopedFile::Mode::Create);
+
+    // Note. Method LocalCountryFile::GetSize() is called below in TEST_EQUAL() macro.
+    // This method checks the size of the content |testMapFile| and |testRoutingFile|.
+    // That means words "map" and "routing".
+    ScopedFile testMapFile(mapFileName, "map");
```

Comment:
s/"map"/mapFileContents/
and remove the comment

Reply:
what's wrong with comment?

Reply:
It will be useless once the code is
```
TEST_EQUAL(mapFileContents.size(), localFile.GetSize(MapOptions::Map), ());
```

Reply:
Besides, it is misleading. Any 3-letter and 7-letter words would do, not just "map" and "routing".